crates/router/src/core/webhooks/incoming.rs

```diff
@@ -1456,34 +1439,27 @@ async fn mandates_incoming_webhook_flow(
                 merchant_context.get_merchant_account().get_id(),
                 &mandate_id,
                 storage::MandateUpdate::StatusUpdate { mandate_status },
-                mandate,
+                mandate.clone(),
                 merchant_context.get_merchant_account().storage_scheme,
             )
             .await
             .to_not_found_response(errors::ApiErrorResponse::MandateNotFound)?;
-        let mandates_response = Box::new(
-            api::mandates::MandateResponse::from_db_mandate(
-                &state,
-                merchant_context.get_merchant_key_store().clone(),
-                updated_mandate.clone(),
-                merchant_context.get_merchant_account(),
-            )
-            .await?,
-        );
+
         let event_type: Option<enums::EventType> = updated_mandate.mandate_status.foreign_into();
         if let Some(outgoing_event_type) = event_type {
-            Box::pin(super::create_event_and_trigger_outgoing_webhook(
+            let primary_object_created_at = Some(mandate.created_at);
+
+            Box::pin(super::add_bulk_outgoing_webhook_task_to_process_tracker(
                 state,
-                merchant_context,
-                business_profile,
+                &business_profile,
+                &mandate_id,
                 outgoing_event_type,
-                enums::EventClass::Mandates,
-                updated_mandate.mandate_id.clone(),
-                enums::EventObjectType::MandateDetails,
-                api::OutgoingWebhookContent::MandateDetails(mandates_response),
-                Some(updated_mandate.created_at),
+                diesel_models::enums::EventClass::Mandates,
+                diesel_models::enums::EventObjectType::MandateDetails,
+                primary_object_created_at,
             ))
-            .await?;
+            .await
+            .map_err(|e| e.change_context(errors::ApiErrorResponse::WebhookProcessingFailure))?;
```

Comment:
Same comments as before, about the `enums::` qualification and the unnecessary `map_err()`.